VERSION

```diff
@@ -1 +1 @@
-0.4.1
+0.4.2
```

Comment:
version increases should happen in master, otherwise the commit + tag that bumped this version won't correlate to the actual code that was published (since we only build + publish from master)